client/actions/auth/index.js

```diff
@@ -29,6 +29,24 @@ export default class AuthActionCreator {
     }
   }
 
+  static doLogin (email, password) {
+    return async (dispatch, getState) => {
+      try {
+        const state = getState()
+        const apiOptions = apiOptionsFromState(state)
+        const user = await usersApiClient.login(apiOptions, email, password)
+        if (!user.hasOwnProperty('error')) {
+          dispatch(AuthActionCreator.login(user))
+          dispatch(push('/'))
```

Comment:
Should the router push happen on the component-end? That way, we can utilize `history.push()` and distinguish more easily between state changes and navigation.